en/micro-integrator/docs/develop/generate-docker-image.md

```diff
@@ -4,52 +4,102 @@ See the topics given below.
+    -  **Generate Docker Image with the Embedded MI**
+
+        1.  Select this option to generate a Docker image with the embedded Micro Integrator runtime of WSO2 Integration studio. 
+
+            !!! Note
+                This is recommended only for testing.
+
+        2.  Click **Next** and enter the following details:
+            
+            ![Create docker image dialog](../assets/img/create_project/generate_docker_image_dialog.png) 
+
+            <table>
+                <tr>
+                    <th>
+                        Parameter
+                    </th>
+                    <th>
+                        Description
+                    </th>
+                </tr>
+                <tr>
+                    <td>
+                        Name of the Application
+                    </td>
+                    <td>
+                        The name of the composite
+                        application with the artifacts created for your ESB project.
+                        The name of the ESB project is displayed by default, but it
+                        can be changed if required.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Application Version
+                    </td>
+                    <td>
+                        The version of the composite
+                        application.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Name of the Docker Image
+                    </td>
+                    <td>
+                        Give a name for the Docker image.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Docker Image Tag
+                    </td>
+                    <td>
+                        A tag for the Docker image to be used
+                        for reference. This is optional.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Export Destination
+                    </td>
+                    <td>
+                        Browse for the preferred location
+                        in your machine to export the Docker image.
+                    </td>
+                </tr>
+            </table>
+
+        3.  Click **Next**. Select the **Config** projects that you want to include in the Docker image and click **Finish**.  
+
+            ![Create docker image](../assets/img/create_project/select_artifact_docker.png)  
+        
+            Once the Docker image is successfully created, a message similar to the following appears in your screen.  
+
+            ![Create docker image](../assets/img/create_project/docker_image_successful.png)
+
+    -   **Generate Docker Image with an Existing Project**
```

Comment:
@rosensilva - When I tried this, nothing happened. Is there suppose to be some notification? Is this similar to the 'Build' option for the 'Docker Exporter Project'?

Reply:
I might take some time in the first time, did you wait about 10 mins?

Reply:
Didn't wait for 10 minutes. :) But now I added an image with the expected notification. I am guessing the docker image will get created and we will receive the 'Docker image successfully created' message?